proto/dydxprotocol/accountplus/accountplus.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package dydxprotocol.accountplus;
+
+option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/accountplus/types";
+
+// Account State
+message AccountState {
+  string address = 1;
+  TimestampNonceDetails timestamp_nonce_details = 2;
+}
+
+// Timestamp nonce details
+message TimestampNonceDetails {
+  // unsorted list of n most recent timestamp nonces
+  repeated uint64 timestamp_nonces = 3;
+
+  // most recent timestamp nonce that was ejected from list above
+  uint64 latest_ejected_nonce = 4;
```

Comment:
Did we intentionally move away from the [spec](https://www.notion.so/dydx/Timestamp-Nonces-dca694b0f8464c3bbf99689ba7e14961?pvs=4#08239425f8264f4891d528992aafba42)? What's the context/ 
```
  // The maximum value of any nonces removed from the above list.
  uint64 max_ejected_nonce = 2;
```

Reply:
I believe the types are the same. As for the naming, I felt like these names were a bit more descriptive. But I have no issue with using the orignal names from the spec.

Reply:
will change back to max